pkg/serverless/aws/arn.go

```diff
@@ -115,3 +160,89 @@ func RestoreCurrentStateFromFile() error {
 	SetRequestID(restoredState.CurrentReqID)
 	return nil
 }
+
+// BuildFunctionARNFromEnv reconstructs the function arn from what's available
+// in the environment.
+func BuildFunctionARNFromEnv() string {
+	region := os.Getenv(regionEnvVar)
+	functionName := os.Getenv(functionNameEnvVar)
+	alias := os.Getenv(aliasEnvVar)
+	accountID := fetchAccountID()
+	arnPrefix := "arn:aws"
+
+	if len(accountID) == 0 || len(region) == 0 || len(functionName) == 0 {
+		return ""
```

Comment:
Would it be more idiomatic to return an error if we can't determine the function ARN, and let the caller handle that?

Reply:
That way, we could optionally log an error message, for example.

Reply:
Good point